avocado/core/version.py

```diff
@@ -16,11 +16,14 @@
 
 __all__ = ['MAJOR', 'MINOR', 'VERSION']
 
+import pkg_resources
```

Comment:
I suppose this is a requirement for the `install_requires` change in the next commit, right?